beebop/utils.py

```diff
@@ -134,6 +191,38 @@ def get_lowest_cluster(clusters_str: str) -> int:
     return min(clusters)
 
 
+def replace_merged_component_filenames(network_folder: str) -> None:
```

Comment:
As discussed, it would be worth clarifying with Nick what he wants to happen in cases where we have samples belonging to different permutations of the same cluster e.g. "1", "1;2", "1;3" - because here we're taking the first cluster 1 file we find, this means that different orderings of samples in a project would potentially lead to different subgraphs being shown for cluster 1, which may or may not include reference samples from the borderlands of clusters 2 and 3.

Reply:
.have made changes as per commit --> https://github.com/bacpop/beebop_py/pull/53/commits/a72f81ff237ac61866fa0d1af7769a285c665b25 .

Now we also store the raw number in the results, so getting the network file we can get the direct file name... thus removing need to replace merged files